python/GafferUITest/PathListingWidgetTest.py

```diff
@@ -1153,14 +1162,18 @@ def headerData( self, canceller = None ) :
 				return self.CellData( value = "Title" )
 
 		with GafferUI.Window() as window :
-			GafferUI.PathListingWidget(
+			widget = GafferUI.PathListingWidget(
 				Gaffer.DictPath( { "a" : 10 }, "/" ),
 				columns = [
 					GafferUI.PathListingWidget.defaultNameColumn,
 					SleepingColumn(),
 				]
 			)
 
+		# Sort by SleepingColumn as sorting by defaultNameColumn doesn't
+		# trigger a data update.
+		widget._qtWidget().header().setSortIndicator( 1, QtCore.Qt.AscendingOrder )
```

Comment:
I think the intention in this test was that showing the window would cause Qt to query the data so it could be displayed. changing to `waitForIdle( 1000 )` on L1179 seems to be sufficient to make that happen without messing about with sorting.

Reply:
Fixed in 5f36636, though I needed to go all the way to `waitForIdle( 20000 )` in order for the test to reliably pass on my workstation...